service/k8sclient/kubectl/kubectl.go

```diff
@@ -134,15 +152,16 @@ func lookupCorrectKubectlCmd(kubectlCmdNames []string) ([]string, error) {
 	}
 
 	// if none found (pass empty kubectlCmdNames) use default version of kubectl.
-	return strings.Split(defaultKubectl, " "), nil
+	return defaultKubectl, nil
 }
 
 // getVersions gets kubectl and Kubernetes cluster version.
-func getVersions(ctx context.Context, kubectlCmd []string) ([]byte, error) {
-	versionsJSON, err := run(ctx, kubectlCmd, []string{"version", "-o", "json"}, nil)
+func getVersions(ctx context.Context, kubectlCmd []string, kubeconfigPath string) ([]byte, error) {
+	versionsJSON, err := run(ctx, kubectlCmd, []string{"version", fmt.Sprintf("--kubeconfig=%s", kubeconfigPath), "-o", "json"}, nil)
```

Comment:
:+1: I also fixed it on my branch.

Reply:
Isn't it weird that we can pass flag with empty value?

Reply:
it should not pass so far if "kubeconfig" is empty:
https://github.com/percona-platform/dbaas-controller/blob/c8ff486b9a477a9999c5d5c2c52ebeedf9c24944/service/k8sclient/kubectl/kubectl.go#L64-L70

Reply:
ok